sql/driving_distance/_withPointsDD.sql

```diff
@@ -34,7 +34,7 @@ CREATE FUNCTION _pgr_withPointsDD(
     distance FLOAT,
 
     directed BOOLEAN DEFAULT true,
-    driving_side CHAR DEFAULT 'b',
+    driving_side CHAR DEFAULT 'r',
```

Comment:
I still like the default value to be `r`, About getting rid of `b` we need to discuss